content/api/commands/contains.md

```diff
@@ -343,6 +343,22 @@ yielded before the `.contains()`.
 cy.get('#main').contains('Jane Lane')
 ```
 
+### Default `<input type="submit">` labels
+
+When `value` is omitted in a `<input type=submit>`, the default label is used
```

Comment:
```suggestion
When the `value` attribute is omitted from an `<input type="submit">`, the default label is used
and can be locale-dependent.
[More info at MDN.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit#omitting_the_value_attribute)

When this happens, the `value` is an empty string, and there is no
programmatic way for Cypress to filter elements by the label displayed by the
user agent. This can cause unexpected failures when using `cy.contains()`
with submit buttons.

The solution in this case is to:

```js
 // assert the empty string
 cy.get('input').should('have.value', '')
 
 // ---or---
 
 // if possible, set the `value` attribute
 <input type=submit value="Submit" />
 ```
```

Reply:
edb7a9274